lib/backend/k8sv2/resources/customnoderesource.go

```diff
@@ -0,0 +1,461 @@
+// Copyright (c) 2017 Tigera, Inc. All rights reserved.
```

Comment:
Does anything use this anymore?

I believe this isn't required now we have FelixConfiguration, BGConfiguration, BGPPeer and ClusterInformation being standard CRDs.

Reply:
Oops.. I committed this by misted.. Removed.